src/planscape/config/conditions.json

```diff
@@ -708,44 +1276,74 @@
               "filepath": "sierra_nevada/forest_resilience/forest_composition",
               "normalized_layer": "sierra-nevada:forestShrubResilTranslate_composition",
               "normalized_data_download_path": "sierra-nevada/forestShrubResilTranslate/composition.tif",
-              "display_name": "Forest Composition",
+              "display_name": "Composition",
               "operation": "MEAN",
               "metrics": [
                 {
                   "metric_name": "seral_stage",
                   "filepath": "sierra_nevada/forest_resilience/forest_composition/late_SeralStage_prop_300m",
-                  "raw_layer": "sierra-nevada:forestShrubResil_Tier2_late_SeralStage_prop",
-                  "raw_data_download_path": "sierra-nevada/forestShrubResil/Tier2/late_SeralStage_prop.tif",
+                  "raw_layer": "sierra-nevada:forestShrubResil_Tier2_SeralStage_EML_2021",
+                  "raw_data_download_path": "sierra-nevada/forestShrubResil/Tier2/SeralStage_EML_2021.tif",
                   "normalized_layer": "sierra-nevada:forestShrubResilTranslate_Tier2_late_SeralStage_prop_scored",
```

Comment:
I just realized, the conditions that don't have a normalized layer still show up as options under the normalized section, but just show nothing
<img width="1713" alt="Screenshot 2023-06-29 at 4 36 41 PM" src="https://github.com/OurPlanscape/Planscape/assets/18537927/7284a542-5dc6-47d4-bf40-07be78e895a0">